e3db-crypto-android/build.gradle

```diff
@@ -51,7 +51,8 @@ dependencies {
     androidTestImplementation 'com.android.support.test:runner:1.0.0'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.0'
     compileOnly project(':e3db-crypto-interface')
-    api 'com.github.joshjdevl.libsodiumjni:libsodium-jni-aar:1.0.7'
+    api("com.goterl.lazycode:lazysodium-android:2.5.0@aar")
```

Comment:
Do we need parens and double quotes? This is the only dependency that uses either.